File: trunk/infoclin/app/models/Usuario.java
package models;

import java.io.Serializable;
import play.db.jpa.Model;

public class Usuario extends Model implements Serializable {
	
	int id;
	public String senha;
	public String login;
	public String usuario;
	public Endereco endereco;
	public String nome;
	public String cpf;
	
	public Usuario(String senha, String usuario, Endereco endereco,
			String nome, String cpf) {
		this.senha = senha;
		this.usuario = usuario;
		this.endereco = endereco;
		this.nome = nome;
		this.cpf = cpf;
	}

}

File: trunk/infoclinDesktop/WindowsFormsApplication1/Dominio/Usuario.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Dominio
{
    public class Usuario
    {
        //int Id; vem do web service
        string Login ;
        string Senha ;
        Endereco Endereco ;
        string Nome ;
        string Cpf;

        public Usuario()
        {
            
        }
        
        public Usuario(/*int id,*/ string login, string senha, Endereco endereco,
            string nome, string cpf)
        {
            //Id = id;
            Login = login;
            Senha = senha;
            Endereco = endereco;
            Nome = nome;
            Cpf = cpf;
        }

        public void conectar()
        {

        }

        public string getNome()
        {
            return Nome;
        }
    }
}
